src/components/model/post/PostForm/index.tsx

```
@@ -27,29 +26,26 @@
 };
 
 export function PostForm({ bottomButtons, textarea, formState }: Props) {
-  const { data: currentUser } = trpc.user.getCurrentUser.useQuery();
-  const isAdmin = currentUser?.planSubscription?.plan.name === "admin";
   // テキストエリア操作のカスタムフック
   const { textareaRef, insertMarkdownAtCursor, insertAtCursor } =
```

ESLint

[error] 'insertMarkdownAtCursor' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)